terms.html

```diff
@@ -3,14 +3,23 @@
 </p>
 
 <dl class="termlist">
+  <dt><dfn data-lt="author|authors|authored|author's|author(s)|authoring">author</dfn></dt>
+  <dd>
+A role that a <a>party</a> performs as it expresses an assertion about a particular <a>entity</a>, i.e., creates a <a>claim</a>.
+  </dd>
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-An assertion made about a <a>subject</a>.
+A digital representation of an assertion made about an <a>entity</a> by a specific <a>party</a>.
+The entity is called the <a>subject</a> of the claim. 
+The <a>party</a> is referred to as the <a>author</a> of the claim, i.e. the one that has expressed the claim.
+The meaning of the claim (its semantics) is determined by its <a>author</a>.
+Different <a>authors</a> can reuse the same semantics for claims through the
+use of author-independent, shared vocabularies.
   </dd>
   <dt><dfn data-lt="credential|credentials">credential</dfn></dt>
   <dd>
-A set of one or more <a>claims</a> made by an <a>issuer</a>. 
-The <a>claims</a> in a credential can be about different <a>subjects</a>.
+A set of one or more <a>claims</a> <a>authored</a> by a single <a>party</a>, that is constructed for the purpose of transferring it to some other <a>party</a>. 
+The <a>claims</a> in a <a>credential</a> can be about different <a>subjects</a>.
```

Comment:
```suggestion
The <a>claims</a> in a <a>verifiable credential</a> can be about different <a>subjects</a>.
```

The group is largely removing commentary about a "credential", since what the group is defining is a "verifiable credential".